src/code/z_camera.c

```diff
@@ -3716,260 +3702,248 @@ s32 Camera_Battle1(Camera* camera) {
-        } else if (temp_v0_6 > 0x2AA8) {
-            temp_v0_6 = 0x2AA8;
+        if (sp8A < -0x2AA8) {
+            sp8A = -0x2AA8;
+        } else if (sp8A > 0x2AA8) {
+            sp8A = 0x2AA8;
         }
 
-        spBC.pitch = Camera_ScaledStepToCeilS(temp_v0_6, sp94.pitch, 1.0f / camera->pitchUpdateRateInv, 5);
-        OLib_AddVecGeoToVec3f(sp48, sp4C, &spBC);
-        spC4 = *sp48;
+        spBC.pitch = Camera_ScaledStepToCeilS(sp8A, atToEyeNextDir.pitch, 1.0f / camera->pitchUpdateRateInv, 5);
+        OLib_AddVecGeoToVec3f(eyeNext, at, &spBC);
+        spC4.pos = *eyeNext;
+
         if (camera->status == CAM_STATUS_ACTIVE) {
             if (!(roData->interfaceFlags & BATTLE1_FLAG_4)) {
-                if ((camera->play->envCtx.skyboxDisabled == 0) || (roData->interfaceFlags & BATTLE1_FLAG_0)) {
-                    if (func_800CBC84(camera, sp4C, &spC4, 0) != 0) {
+                if (!camera->play->envCtx.skyboxDisabled || (roData->interfaceFlags & BATTLE1_FLAG_0)) {
+                    if (func_800CBC84(camera, at, &spC4, 0) != 0) {
                         rwData->unk_1A |= 0x1000;
                     } else {
                         rwData->unk_1A &= ~0x1000;
                     }
                 } else if (roData->interfaceFlags & BATTLE1_FLAG_1) {
-                    func_800CBC84(camera, sp4C, &spC4, 3);
+                    func_800CBC84(camera, at, &spC4, 3);
                 } else {
-                    OLib_Vec3fDistNormalize(&sp114, sp4C, &spC4);
-                    spC4.x -= sp114.x;
-                    spC4.y -= sp114.y;
-                    spC4.z -= sp114.z;
+                    OLib_Vec3fDistNormalize(&sp114, at, &spC4.pos);
+                    spC4.pos.x -= sp114.x;
+                    spC4.pos.y -= sp114.y;
+                    spC4.pos.z -= sp114.z;
                 }
-                *sp50 = spC4;
-                ;
+                *eye = spC4.pos;
                 rwData->unk_1A &= ~0x10;
-            } else if ((camera->play->envCtx.skyboxDisabled == 0) || (roData->interfaceFlags & BATTLE1_FLAG_0)) {
-                if (func_800CBC84(camera, sp4C, &spC4, 0) != 0) {
+            } else if (!camera->play->envCtx.skyboxDisabled || (roData->interfaceFlags & BATTLE1_FLAG_0)) {
+                if (func_800CBC84(camera, at, &spC4, 0) != 0) {
+                    s16 screenX;
+                    s16 screenY;
+
                     rwData->unk_1A |= 0x1000;
-                    spF8 = OLib_Vec3fDist(sp4C, sp8C);
-                    spF4 = OLib_Vec3fDist(sp4C, &spC4);
-                    phi_f0_3 = (rwData->unk_1A & 0x10) ? 40.0f : 0.0f;
-                    spF8 = spF8 + phi_f0_3;
-                    Actor_GetScreenPos(camera->play, camera->focalActor, &sp5E, &sp5C);
-                    if ((spF4 < spF8) || ((sp5E >= 0) && (sp5E < 0x141) && (sp5C >= 0) && (sp5C < 0xF1))) {
+                    spF8 = OLib_Vec3fDist(at, &sp8C->pos);
+
+                    //! FAKE:
+                    if (1) {}
```

Comment:
When I have a fakematch like this, I tend to put a decomp.me link in there *without* this fake code in here to show what it looks like without it, and to give people an opportunity to try and match it without the fake section. However, I just sort of decided that on my own without consulting anyone about it. What do you think about adding such a link here?

Reply:
MM's Camera is currently sitting at 29 fake matches, so that would be a lot of scratches 😅. Me personally, I always tend to mess around with fakes in-code, so I never use scratches in code, but I'm not opposed if others want to add it as we go

Reply:
I also find scratches can become desynced with docs over time, especially looking back at scratches in-code that were put in half-a-year ago

Reply:
Ahh, that's a bit of context I missed. Since camera has a *lot* of fake matches, I suppose it makes sense to skip this, no worries! Like I said, I didn't consult anyone before I started doing this, so it's not like this is a widely-done thing or anything.